.github/workflows/go.yaml

```diff
@@ -25,17 +25,9 @@ jobs:
       - name: Install python3 deps
         run: pip3 install requests
 
-      - name: Cache tools
```

Comment:
We still want to cache at least go dependencies. Let checksum of install-dependencies.sh be a cache key. You only have to find out where these deps are stored, probably GOBIN. This speeds up CI builds.